haas/api.py

```diff
@@ -505,21 +563,73 @@ def headnode_detach_network(headnode, hnic):
                             # Network Code #
                             ################
 
+@rest_call('GET', '/networks')
```

Comment:
Could you add a `Schema` argument, similar to this:
```python
@rest_call('GET', '/projects', Schema({}))
def list_projects():
```

Now that we've merged #596, all calls should have a `Schema`.